lib/pure/times.nim

```diff
@@ -522,6 +523,35 @@ proc getDaysInYear*(year: int): int =
     doAssert getDaysInYear(2001) == 365
   result = 365 + (if isLeapYear(year): 1 else: 0)
 
+proc getWeeksInYear*(y: int): YearweekRange =
```

Comment:
Use `Natural` and/or document that the argument must be positive, same with other proc. 
Also needs `{.since.}` and changelog entry.

Reply:
Thanks for the help!

Reply:
@juancarlospaco  > Use Natural

Added unit tests that demonstrate support for iron age dates instead (use case for negative integer)

since/changelog coming up...